extensions/binding/org.eclipse.smarthome.binding.hue/README.md

```diff
@@ -125,6 +127,8 @@ sitemap demo label="Main Menu"
 		Slider item=		Light1_Dimmer
 		Colorpicker item=	Light1_Color
 		Slider item=		Light1_ColorTemp
+		Switch item=	    Light1_Alert mappings=[NONE="None", SELECT="Alert", LSELECT="Long Alert"]
```

Comment:
the "old" text used tabs here - I do not really care whether its spaces or tabs in the readme, but it should be consistent.